descriptor.md

```diff
@@ -76,20 +76,17 @@ hex         := /[a-f0-9]+/
 
 Some example digest strings include the following:
 
-digest                                                                  | algorithm           |
+digest string                                                           | algorithm           |
```

Comment:
With 23af834 moving us towards consistently using [digest/algorithm (identifier)/hex][1] matching our ABNF, I think we may want to stick to “digest” instead of using “digest string”.  I don't mind if existing instances of “digest string” are changed to “digest” in this PR or not, but I think this line (and the “Before consuming…” line below) should be left alone in this PR.

[1]: https://github.com/opencontainers/image-spec/blame/52d4836391a210e5af79afd70cadd8643829f325/descriptor.md#L72